source/default/_patterns/00-protons/non-printing/_spacing.scss

```diff
@@ -1,7 +0,0 @@
-// Custom spacing variables and mixins
```

Comment:
@illepic 
Consider this PR removes this and replaces with Spacing from Bootstrap, should we be demonstrating an override in Bootstrap Overrides? I don't think so, but want to check in.

Reply:
@jryanconklin @illepic -- I like that idea and added a demo of a few spacing overrides in _bootstrap-overrides.scss. Also, since the overrides must be defined before the imports, I had to use 1rem in place of $spacer. Please review those changes when you get a chance. [496ec3b](https://github.com/phase2/particle/pull/474/commits/496ec3be020c5442ac7e483cd10559d8b15b68e8)